src/components/Documents/DocumentTable.jsx

```diff
@@ -15,15 +15,20 @@ import { StyledTableCell } from '../Table/TableStyles';
 import DocumentTableRow from './DocumentTableRow';
 import { EmptyListNotification, LoadingAnimation } from '../Notification';
 
+/**
+ * @typedef {import("../../typedefs.js").documentTableProps} documentTableProps
```

Comment:
To see how to create the typedef for the React prop, you could look at one of the prop definitions in `typedefs.js` (see `footerProps` in Line 287).

As you would see there, `@property` is a decorator that help define objects in JSDoc, namely for the object being described within the JSDoc.